arm-dns/2016-04-01/swagger/dns.json

```diff
@@ -431,8 +430,9 @@
             "name": "$top",
             "in": "query",
             "required": false,
-            "type": "string",
-            "description": "Query parameters. If null is passed returns the default number of zones."
+            "type": "integer",
+            "format": "int32",
+            "description": "Query parameters. If not specified returns the default number of recordsets."
```

Comment:
@vladrai - What is that default number? Is it possible to specify that value in the description?